packages/world/src/modules/keysintable/KeysInTableHook.sol

```diff
@@ -8,61 +8,61 @@ import { KeysInTable } from "./tables/KeysInTable.sol";
 import { UsedKeysIndex } from "./tables/UsedKeysIndex.sol";
 
 /**
- * Note: if a tableId with composite keys is used, only the first key is indexed
+ * Note: if a tableId with composite keys is used, only the first five keys of the tuple are indexed
  */
 contract KeysInTableHook is StoreHook {
-  function handleSet(bytes32 tableId, bytes32[] memory key) internal {
-    bytes32 keysHash = keccak256(abi.encode(key));
+  function handleSet(bytes32 tableId, bytes32[] memory keyTuple) internal {
+    bytes32 keysHash = keccak256(abi.encode(keyTuple));
 
-    // If the key has not yet been set in the table...
+    // If the keyTuple has not yet been set in the table...
     if (!UsedKeysIndex.getHas(tableId, keysHash)) {
       uint40 length = uint40(KeysInTable.lengthKeys0(tableId));
 
-      // Push the key to the list of keys in this table
-      if (key.length > 0) {
-        KeysInTable.pushKeys0(tableId, key[0]);
-        if (key.length > 1) {
-          KeysInTable.pushKeys1(tableId, key[1]);
-          if (key.length > 2) {
-            KeysInTable.pushKeys2(tableId, key[2]);
-            if (key.length > 3) {
-              KeysInTable.pushKeys3(tableId, key[3]);
-              if (key.length > 4) {
-                KeysInTable.pushKeys4(tableId, key[4]);
+      // Push the keyTuple to the list of keys in this tableId
```

Comment:
```suggestion
      // Push the keyTuple to the list of keys in this table
```

Reply:
I think `keyTuple` is correct in these cases